aries-site/src/components/cards/ContentCard.js

```diff
@@ -6,15 +6,15 @@ import { Identifier, Tile } from 'aries-core';
 import { PreviewImageCard } from './PreviewCard';
 import { useDarkMode } from '../../utils';
 
-const StyledTile = styled(Tile)`
+export const StyledTile = styled(Tile)`
```

Comment:
Why does `StyledTile` need to be exported? Where is it being consumed besides `ContentCard`?

Reply:
It's being used in `ContentPreviewCard` which are the layout previews on the page-layouts page. I created a new component because the content of these didn't have the same data structure as the typical cards on the site. As opposed to adding a bunch of inline conditionals, I thought it might be more simple and clear for it to be its own component.